packages/client/src/svelte/modules/systems/createComponentSystem.ts

```diff
@@ -15,7 +15,7 @@ export function createComponentSystem(componentKey: string) {
         : nextValue
 
     const entityID = update.entity
-    const propertyName = toCamelCase(componentKey)
+    const propertyName = componentKey === "ContainedMaterial" ? "materialId" : toCamelCase(componentKey)
```

Comment:
I realized way too late that table names were being turned into client keys
This is a hacky solution which hopefully works, up to you how to best resolve this (we could refactor type name to something other than MaterialId for example, or just leave this hack)